cmake/scripts/windows/ArchSetup.cmake

```diff
@@ -12,6 +12,7 @@ endif()
 # -------- Paths (mainly for find_package) ---------
 
 set(PLATFORM_DIR platform/win32)
+set(APP_RENDER_SYSTEM dx11)
```

Comment:
please add the same for `cmake/scripts/windowsstore/ArchSetup.cmake`

Reply:
done, thought they shared stuff..

Reply:
thanks, not all things are shared.